horizomer/utils/tests/test_tree.py

```diff
@@ -16,7 +16,8 @@
 
 from horizomer.utils.tree import (
     support, unpack, has_duplicates, compare_topology, intersect_trees,
-    unpack_by_func, read_taxdump, build_taxdump_tree, order_nodes)
+    unpack_by_func, read_taxdump, build_taxdump_tree, order_nodes,
+    is_nodes_ordered)
```

Comment:
Shouldn't it be `is_ordered` or `are_nodes_ordered`? I would suggest former because the input variable is a tree.